std/algorithm/iteration.d

```diff
@@ -2029,9 +2029,11 @@ version(none) // This requires support for non-equivalence relations
 /**
 Lazily joins a range of ranges with a separator. The separator itself
 is a range. If a separator is not provided, then the ranges are
-joined directly without anything in between them (often called $(D flatten)
+joined directly without anything in between them (often called `flatten`
 in other languages).
 
```

Comment:
Looks a bit better IMO if you merge these two clauses, mentioning both alternate names in one go. Nothing what would make me reject this trough.